english/metallica/metallica.md

```diff
@@ -1,8 +1,13 @@
-Metallica Wiki? [here](https://en.wikipedia.org/wiki/Metallica)
+You are in for a treat!
+ (giggles)
+ 
+Brief history (Wikipedia): 
+Metallica is an American heavy metal band. The band was formed in 1981 in Los Angeles, California, by drummer Lars Ulrich and vocalist/guitarist James Hetfield, and has been based in San Francisco, California for most of its career. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.